Sources/BlogModuleApi/Objects/BlogAuthor/BlogAuthorListObject.swift

```diff
@@ -10,15 +10,27 @@ import Foundation
 public struct BlogAuthorListObject: Codable {
 
     public var id: UUID
-    public var name: String
+    public var name: String?
```

Comment:
Why is the name optional? Maybe I'm missing something... but imho it should always have a value.

Reply:
For this, actually this is in the RestAPI return. The idea, is when it is deleted, you just return the timestamp, and ID, in ou case we don't return any more data, not even the title